src/index.js

```diff
@@ -14,7 +14,9 @@ const mapToEllipse = ({ x, y }, rx, ry, cosphi, sinphi, centerx, centery) => {
 }
 
 const approxUnitArc = (ang1, ang2) => {
-  const a = 4 / 3 * Math.tan(ang2 / 4)
+  // See http://spencermortensen.com/articles/bezier-circle/ for the derivation
+  // of this constant.
+  const c = 0.551915024494;
```

Comment:
Great thanks! It's failing CI on linting here (semicolon). Apart from that I'm ready to accept.

Reply:
No problem. I can make that change.